aws/Full-PAS-Deployment.yaml

```diff
@@ -950,33 +1090,6 @@ Resources:
       ResourceSignal:
         Timeout: PT10M
     DeletionPolicy: Retain
-  InstancesSSMPolicy:
-    Type: 'AWS::IAM::Policy'
-    Properties:
-      PolicyName: InstancesSsmAccess
-      PolicyDocument:
-        Statement:
-          - Effect: Allow
-            Action:
-              - 'ssm:GetParameters'
-            Resource:
-              - !Sub 'arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:parameter/*'
-      Roles:
-        - !Ref ComponentInstanceRole
```

Comment:
Component Instances shouldn't have access to SSM to receive vault password?

Reply:
its leftovers from from 11.2 this policies are included as inlines policies at the roles